### ISSUE TITLE
Bump docker version

### ISSUE THREAD

**Issue by pavolloffay** [testcontainers/testcontainers-go#74]
The current pinned docker version is 0.7.3 it would be great to update it. The latest upstream version is 17.03 https://github.com/moby/moby/releases 

https://github.com/testcontainers/testcontainers-go/blob/b130c69aad6475d8decc2794ba45b6c98a316e79/go.mod#L8

Because of the old docker version I am probably getting 
```
# github.com/jaegertracing/jaeger/vendor/github.com/testcontainers/testcontainers-go
vendor/github.com/testcontainers/testcontainers-go/docker.go:190:8: client.NegotiateAPIVersion undefined (type *client.Client has no field or method NegotiateAPIVersion)
vendor/github.com/testcontainers/testcontainers-go/docker.go:327:32: p.client.DaemonHost undefined (type *client.Client has no field or method DaemonHost)
```

**Comment by gianarb** [comment]
That' weird! I will work on it now

**Comment by xtreding** [comment]
@gianarb Hi!
I have the same problem. I've added testcontainers-go v.0.0.4 to my project and it uses methods (
vendor/github.com/testcontainers/testcontainers-go/docker.go:190:8: client.NegotiateAPIVersion undefined (type *client.Client has no field or method NegotiateAPIVersion)
vendor/github.com/testcontainers/testcontainers-go/docker.go:327:32: p.client.DaemonHost undefined (type *client.Client has no field or method DaemonHost)
)
from docker/client which exist only in master branch(https://github.com/moby/moby/blob/master/client/client.go#L205). 
But I can't set my docker/client version to master. How can I solve this problem?

**Comment by gianarb** [comment]
Are you using `go mod` @xtreding ? Why you can not set your docker/client version to master?

**Comment by gianarb** [comment]
Unfortunately this is a known limitation with `go mod` and repositories that do not tag following semver.

I tried this solution @xtreding @pavolloffay but you need to play with your go mod:

https://github.com/testcontainers/testcontainers-go/pull/79

Can you give it a try? You can use `go get github.com/testcontainers/testcontainers-go@33d1f3b`

**Comment by gianarb** [comment]
Fixed here

https://github.com/testcontainers/testcontainers-go/releases/tag/v0.0.5

I a not using docker/engine over moby/moby because it "kind of" use semver

**Comment by yunsubaek** [comment]
> Fixed here
> 
> https://github.com/testcontainers/testcontainers-go/releases/tag/v0.0.5
> 
> I a not using docker/engine over moby/moby because it "kind of" use semver

@gianarb 
Even though upgrade with released version(v0.0.5), same issue is comes out.
Could you please help me to solve this problem?
vendor/github.com/testcontainers/testcontainers-go/docker.go:202:8: client.NegotiateAPIVersion undefined (type *client.Client has no field or method NegotiateAPIVersion)
vendor/github.com/testcontainers/testcontainers-go/docker.go:346:32: p.client.DaemonHost undefined (type *client.Client has no field or method DaemonHost)
